server.py

```diff
@@ -206,8 +250,24 @@ def teardown_request(exception):
 
     # Update the metadata
     metadata = metadata_manager.MetadataManager()
-    metadata.clear_metadata()
-    metadata.update_servers(server_list)
+    metadata.clear_metadata() # shouldn't do this!
```

Comment:
I left the comment for myself to change it later. The thing is that we don't want to clear the metadata every time we spin up the server, so this line should be removed.